packages/react-form/src/utilities.ts

```diff
@@ -205,3 +205,7 @@ export function defaultDirtyComparator<Value>(
     ? !shallowArrayComparison(defaultValue, newValue)
     : defaultValue !== newValue;
 }
+
+export function undirtyFields(fieldBag: FieldBag) {
```

Comment:
I thought about potentially making this a hook (using `useVisitFields`, à la `useReset`) but decided against it for a couple reasons:
- Unlike `useDirty`, `useSubmit`, or `useReset`, "undirtying" fields didn't strike me as "core" form functionality
- Unlike `useReset`, whose memoized callback I can see being passed into child components like buttons, undirtying fields is probably something that would be kept close to submit logic. If consumers really need to pass in a `undirtyFields` callback, they can probably roll their own hook using `useCallback`

I'm pretty new to hooks, so if anyone has thoughts about this please let me know.

Reply:
I think this is the right choice for the utility, though I'd like to see a bound version returned from `useForm` for parity with `reset`.